docs/governance/staking-with-realms.md

```diff
@@ -1,82 +1,69 @@
-<img className="docsheader" src={useBaseUrl('/img/blockchain/realmsheader.png')} />
-
-Like many other Solana Ecosystem projects, Helium utilizes [Realms](https://realms.heliumvote.com/)
-to organize and manage vote escrow tokens, delegation, and more. Stakes can be created and later
-managed within Realms at any time at https://realms.heliumvote.com/.
+Helium utilizes [Helium Vote](https://heliumvote.com), a native application built by the Helium 
+Foundation to organize and manage vote escrow tokens, delegation, and more. Stakes can be created 
+and later managed within Helium Vote at any time at https://heliumvote.com.
 
 ## Creating a Stake
 
-Anyone with HNT in a wallet can create a stake using Realms either using an in-wallet browser or a
-browser-connected wallet.
+Anyone with HNT, MOBILE or IOT in their Helium Wallet App or Solana-compatible browser wallet
+can be used (such as Phantom orBackpack).
 
 While staking IOT or MOBILE tokens is possible through their respective subDAOs, those positions are
 only eligible for voting power and cannot currently be delegated for the purposes of earning
 rewards.
 
-This guide outlines using Realms with the [Helium Wallet App](/wallets/helium-wallet-app). To use
-Realms on a desktop browser, a Solana-compatible browser wallet can be used (such as Phantom or
-Backpack).
+This guide outlines using Helium Vote with the [Helium Wallet App](/wallets/helium-wallet-app). To use
+Helium Vote on a desktop browser, a Solana-compatible browser wallet can be used (such as Phantom or
+Backpack) by navagating to https://heliumvote.com.
 
-The Helium Wallet App's browser can be accessed via the globe icon in the bottom navigation.
 
-<figure className="screensnippet-wrapper">
-  <img src={useBaseUrl('/img/realms/realmsbrowser.png')} />
-  <figcaption>
-    veHNT can be managed via <code>https://realms.heliumvote.com/dao/hnt</code>
-  </figcaption>
-</figure>
-<br />
+### Staking in the Helium Wallet App
 
-### Signing Into Realms
-
-Connect your account to Realms using the in-app browser from the Helium Wallet App.
+Staking in the Helium Wallet App can be done by navigating to the <code>Governance</code> tab within the Helium Wallet.
+Ensure you select the token you want to stake (HNT, MOBILE or IOT) at the top.
 
 <figure className="screensnippet-wrapper">
-  <img src={useBaseUrl('/img/realms/connectaccount.png')} />
+  <img src={useBaseUrl('https://github.com/helium/docs/pull/1784#issuecomment-1933058646')} />
   <figcaption>
-    This user is logged in with their <code>4qPt5...</code> account.
+   The Governance tab is the fourth tab from the left. The HNT stake button at the top is the default selection.
   </figcaption>
 </figure>
 <br />
 
-Once your wallet has been connected to Realms, your token balances (or existing positions) will be
-displayed in the governance section of the DAO.
+Once in the <code>Governance</code> tab, you will see <code>Your Voting Power</code>. Click <code>Your Voting Power</code>
```

Comment:
for code formatting, markdown allows using the `  wrapper

Reply:
It is advised to not over-use it as it can impact the reading experience.

Reply:
Noted. Removed it